### PR TITLE
KeysetPaginator more test

### PR DESCRIPTION
**Pull request by kamarton**
KeysetPaginator more test.

| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | -

### CODE REVIEW

File: tests/Paginator/KeysetPaginatorTest.php [hunk 1]
```diff
@@ -169,4 +195,29 @@ public function testReadSecondPage(): void
 
         $this->assertSame($expected, $this->iterableToArray($paginator->read()));
     }
+
+    public function testReadSecondPageWithOtherOrderThanLastValueField(): void
+    {
+        $sort = (new Sort(['id', 'name']))->withOrderString('name');
+
+        $dataReader = (new IterableDataReader($this->getDataSet()))
```
Comment (samdark) [inline tests/Paginator/KeysetPaginatorTest.php]: Was it meant to be getDataSetSortedByName()?

Comment (demonking) [reply tests/Paginator/KeysetPaginatorTest.php]: Any ordered list by name, i think he mean.
But the function name is ugly :)

Comment (kamarton) [reply tests/Paginator/KeysetPaginatorTest.php]: I am idiot :) Test for sorted by non ID.

Comment (kamarton) [reply tests/Paginator/KeysetPaginatorTest.php]: I removed.

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**